internal/v1/api.yaml

```diff
@@ -382,6 +382,8 @@ components:
       properties:
         image_status:
           $ref: '#/components/schemas/ImageStatus'
+        compose_request:
```

Comment:
In `ComposesResponseItem:` we just call it `request`, so let's call it the same thing?

Also it can be a required property. Adding required properties in responses is fine.

Reply:
Problem is, the JSON Go tag is actually set to `compose_request`:

```
type ComposeStatus struct {
	ComposeRequest *ComposeRequest `json:"compose_request,omitempty"`
	ImageStatus    ImageStatus     `json:"image_status"`
}
```

Is it worth creating a new struct?

Reply:
the JSON go tag is generated based on the name of the field in the openapi spec though.